src/components/display/TokenIcon.js

```diff
@@ -36,10 +36,10 @@ import type { Chain } from 'models/Chain';
 // Assets
 import IconEthereum from 'assets/icons/svg/icon-24-ethereum.svg';
 import IconBinance from 'assets/icons/svg/icon-24-binance.svg';
-import IconXdai from 'assets/icons/svg/icon-24-xdai.svg';
 import IconPolygon from 'assets/icons/svg/icon-24-polygon.svg';
 import IconAvalanche from 'assets/icons/svg/icon-24-avalanche.svg';
 import IconOptimism from 'assets/icons/svg/icon-24-optimism.svg';
+import IconGnosis from 'assets/icons/svg/icon-24-gnosis.svg';
```

Comment:
The native token on Gnsosis is still xDai: https://developers.gnosischain.com/for-users/getting-started-with-gc

Reply:
Is this means here to show the icon as previously as it is?

Reply:
I added an icon for the xDai native token.